client/src/components/app/AddProperties.js

```diff
@@ -16,19 +16,19 @@ export const PropertiesEditor = () => {
                         }
                         return (
                             <>
-                                <label htmlFor={`propertyOptions.${index}.name`}>Property Name: </label>
-                                <Field name={`propertyOptions.${index}.name`} type="text" />
-                                <label htmlFor={`propertyOptions.${index}.type`}>Property Type: </label>
-                                <Field name={`propertyOptions.${index}.type`} as="select">
+                                <strong htmlFor={`propertyOptions.${index}.name`}>Property Name: </strong>
+                                <Field id="fieldMedium" name={`propertyOptions.${index}.name`} type="text" />
+                                <strong htmlFor={`propertyOptions.${index}.type`}>Property Type: </strong>
+                                <Field id="fieldMedium" name={`propertyOptions.${index}.type`} as="select">
                                     {propertyTypeOptions.map(propertyTypeOption => (<option value={propertyTypeOption.value}>{propertyTypeOption.label}</option>))}
                                 </Field>
-                                <label htmlFor={`propertyOptions.${index}.values`}>List of Values:</label>
-                                <Field name={`propertyOptions.${index}.values`} as='textarea' />
-                                <label htmlFor={`propertyOptions.${index}.min`}>Min (length, words, #, etc): </label>
-                                <Field name={`propertyOptions.${index}.min`} type="number" />
-                                <label htmlFor={`propertyOptions.${index}.max`}>Max (length, words, #, etc): </label>
-                                <Field name={`propertyOptions.${index}.max`} type="number" />
-
+                                <strong htmlFor={`propertyOptions.${index}.values`}>List of Values:</strong>
+                                <Field id="fieldLarge" name={`propertyOptions.${index}.values`} />
```

Comment:
I think this should still be a textarea; currently the large field is a heavily spaced single line, whereas a textarea allows multiple lines to be entered